### ISSUE TITLE
listen_start() fails when a client is started

### ISSUE THREAD

**Issue by Harry-LeBlanc** [Matthew-Beliveau/SecureDrop#21]
I have currently fixed this with a try, except block but I don't think this is a permanent fix. The error said: bad file descriptor, I think this might be solvable by the server.detach() function, but it needs to be looked into.
